datalake/src/main/java/com/sequenceiq/datalake/cm/ClouderaManagerRangerUtil.java

```diff
@@ -118,17 +109,61 @@ public boolean isCloudIdMappingSupported(String stackCrn) throws ApiException {
         return isCloudIdMappingSupported(rolesResourceApi, clusterName, rangerUserSyncRoleName);
     }
 
-    public Optional<ApiCommand> setAzureCloudIdentityMapping(String stackCrn, Map<String, String> azureUserMapping) throws ApiException {
+    private Map<String, String> userMappingStrToMap(String userMappingStr) {
+        userMappingStr = userMappingStr.trim();
+        if (userMappingStr.isEmpty()) {
+            return Collections.emptyMap();
+        }
+        Map<String, String> ret = new HashMap<>();
```

Comment:
could you give this variable a meaningful name?

Reply:
Renamed.